pkgs/development/libraries/openmpi/default.nix

```diff
@@ -78,21 +98,11 @@ in stdenv.mkDerivation rec {
    '';
 
   postFixup = ''
-    # default compilers should be indentical to the
-    # compilers at build time
-
-    sed -i 's:compiler=.*:compiler=${targetPackages.stdenv.cc}/bin/${targetPackages.stdenv.cc.targetPrefix}cc:' \
-      $out/share/openmpi/mpicc-wrapper-data.txt
-
-    sed -i 's:compiler=.*:compiler=${targetPackages.stdenv.cc}/bin/${targetPackages.stdenv.cc.targetPrefix}cc:' \
-       $out/share/openmpi/ortecc-wrapper-data.txt
-
-    sed -i 's:compiler=.*:compiler=${targetPackages.stdenv.cc}/bin/${targetPackages.stdenv.cc.targetPrefix}c++:' \
-       $out/share/openmpi/mpic++-wrapper-data.txt
-  '' + lib.optionalString fortranSupport ''
-
-    sed -i 's:compiler=.*:compiler=${gfortran}/bin/${gfortran.targetPrefix}gfortran:'  \
-       $out/share/openmpi/mpifort-wrapper-data.txt
+    # Default compilers should be identical to the compilers at build time
```

Comment:
That throws a lot of warnings now, since it also tries to substitute the symlinked files.

Reply:
Weird. I thought I put ``2>/dev/null`` at the end. I'll fix that.